### PR TITLE
wru 2020 update (2.0.0)

### PR DESCRIPTION
**Pull request by mdblocker**
Changed all defaults to 2020 data, and added a message to notify users of the year being used when pulling census data.

### CODE REVIEW

File: R/get_census_api.R [hunk 1]
```diff
@@ -5,8 +5,8 @@
 #' This function obtains U.S. Census data via the public API. User
 #' can specify the variables and region(s) for which to obtain data.
 #'
-#' @param data_url URL root of the API, including the question mark,
```
Comment (1beb) [inline R/get_census_api.R:8]: Verify this change to docs please. @mdblocker

File: R/get_census_api_2.R [hunk 2]
```diff
@@ -6,8 +6,8 @@
 #' It is used by the \code{get_census_api} function. The user should not need to call this 
 #' function directly.
 #'
-#' @param data_url URL root of the API, including the question mark, 
-#'  e.g., \code{"https://api.census.gov/data/2010/dec/sf1?"}.
```
Comment (1beb) [inline R/get_census_api_2.R:10]: Please verify.

**Review by 1beb — CHANGES_REQUESTED** [review]

File: R/predict_race.R [hunk 3]
```diff
@@ -136,8 +136,8 @@
 #' @export
 
 predict_race <- function(voter.file, census.surname = TRUE, surname.only = FALSE,
-                         surname.year = 2010, census.geo, census.key = NULL, census.data = NULL, age = FALSE,
-                         sex = FALSE, year = "2010", party = NULL, retry = 3, impute.missing = TRUE,
+                         surname.year = 2020, census.geo, census.key = NULL, census.data = NULL, age = FALSE,
```
[the remark this answers is not in the document]
Comment (mdblocker) [reply R/predict_race.R:139]: message added, surname.year removed

File: R/get_census_api.R (hunk 1 above)
Comment (mdblocker) [reply R/get_census_api.R:8]: image
Verified, sends me to the 2020 version of the API-endpoint (screen grab for confirmation)

File: R/get_census_api_2.R (hunk 2 above)
Comment (mdblocker) [reply R/get_census_api_2.R:10]: verified, same 2020 endpoint as the sister R script.

**Review by 1beb — CHANGES_REQUESTED** [review]
Please remove the binary files in src/ they are a remnant of your local build that should not be distributed with the package.

**Comment by mdblocker** [comment]
@beb I rescued the afore mentioned files from the trash and added them to .gitignore instead, added one I missed. I think I have all the files dated to my commit. Unless you see one I missed I believe this is ready to go.

**Comment by 1beb** [comment]
@mdblocker Now that we've updated the defaults we need to update the tests to reflect the updates. What I would like to do is have two separate copies. One copy with the old default arg set and a new set with the updated arg that runs off of a different census data set. You'll need to recreate the data for 2020 in tests/data/census_block_nj_2010 and save it as a new file 2020.

**Pull request merged**